assignment-2/cmmi.md

```diff
@@ -1,2 +1,136 @@
+| DAR - Decision Analysis and Resolution       | Not at all | Not at all | |
+| IPM - Integrated Project Management          | Partially | Fully | |
+| OPD - Organizational Process Definition      | Not at all | Not at all | |
+| OPF - Organizational Process Focus           | Partially | Partially | |
+| OT - Organizational Training                 | Not at all | Not at all | |
+| PI - Product Integration                     | Partially | Fully | |
+| RD - Requirements Development                | Partially | Fully | |
+| RSKM - Risk Management                      | Not at all | Fully | |
+| VAL -  Validation                            | Partially | Fully | |
+| VER -  Verification                          | Partially | Fully | |
+| **Maturity Level 4 - Quantitatively Managed**      |
+| OPP - Organizational Process Performance     | Partially | Not at all | |
+| QPM - Quantitative Project Management        | Not at all | Not at all | |
+| **Maturity Level 5 - Optimizing**                    |
+| CAR - Causal Analysis and Resolution         | Not at all | Partially | |
+| OPM - Organizational Performance Management  | Partially | Not at all | |
+
+## Scrum
+
+| Cmmi Process Area | Covered | Rationale  |
+| ------------------| ------- | ---------  |
+| **Level 2**           |
+| CM                | Not at all | Scrum does not offer any information on the configuration management of work products.            |
+| MA                | Partially | Scrum specifies some possible measurements such as team velocity but not for all of its process areas such direct insight into the quality of the work produced by a product owner.             |
+| PMC               | Partially | Scrum has the notion of sprint goals. This can be used to determine how well a team is performing based on how often they reach their sprint goals. There is also velocity which can be used to plan and determine if any form of attempted process improvement are adding any value. Scrum specifies that the Scrum master is responsible for for making sure that the team performs optimally but does not formally state everything they should do and how it should be done.           |
+| PP                | Fully     | This is corresponds with sprint planning meetings where backlog items are selected and broken down into tasks for the upcoming sprint backlog which are then assigned to team members.             |
+| PPQA              | Partially | Scrum prescribes some practices to measure how processes are performed and can for example be used to by the Scrum master to know when they should aid the team but it says nothing about how about to assure the quality of products produced using it. Most Scrum teams have to incorporate quality assurance into their definition of done for sprint tasks            |
+| REQM              | Fully | The requirements are in user stories in the backlog and are the responsibility of the product owner to determine and formulate.            |
```

Comment:
True. but the scrum requirements don't have much to do with architectutral requirements usually. Just the user functionality